File: Data Structures & Algorithms/implement-queue-using-stacks/submission-3.py
class MyQueue:

    def __init__(self):
        self.q = []
        self.start = 0
        self.size = 0

    def push(self, x: int) -> None:
        self.q.append(x)
        self.size += 1

    def pop(self) -> int:
        
        popped = self.q[self.start]
        self.start += 1

        return popped

    def peek(self) -> int:
        return self.q[self.start]

    def empty(self) -> bool:
        if self.start >= self.size:
            return True
        return False


# Your MyQueue object will be instantiated and called as such:
# obj = MyQueue()
# obj.push(x)
# param_2 = obj.pop()
# param_3 = obj.peek()
# param_4 = obj.empty()

File: Data Structures & Algorithms/design-hashmap/submission-3.cpp
class MyHashMap {
private:

    unordered_map<int, int> hashmap;

public:
    MyHashMap() {}
    
    void put(int key, int value) {
        hashmap[key] = value;
    }
    
    int get(int key) {
        if (hashmap.contains(key)) return hashmap[key];
        return -1;
    }
    
    void remove(int key) {
        hashmap[key] = -1;
    }
};

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap* obj = new MyHashMap();
 * obj->put(key,value);
 * int param_2 = obj->get(key);
 * obj->remove(key);
 */